frontend/src/metabase/dashboard/components/Dashboard/DashboardEmptyState/DashboardEmptyState.tsx

```diff
@@ -54,11 +54,13 @@ export function DashboardEmptyState({
   );
 }
 
-interface TabEmptyStateProps {
+interface DashboardEmptyStateWithoutAddPrompt {
```

Comment:
nit: Do we want to keep "Props" suffix here to differentiate between the type and component?

```suggestion
interface DashboardEmptyStateWithoutAddPromptProps {
```

Reply:
Yup, accidentally forget it